components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -5226,6 +5226,88 @@ public static String getMediationSequenceUuid(String sequenceName, int tenantId,
         return null;
     }
 
+    /**
+     * Returns attributes correspond to the given mediation policy name and direction
+     *
+     * @param policyName name of the  sequence
+     * @param tenantId   logged in user's tenantId
+     * @param direction  in/out/fault
+     * @param identifier API identifier
+     * @return attributes(path, uuid) of the given mediation sequence or null
+     * @throws APIManagementException If failed to get the uuid of the mediation sequence
+     */
+    public static Map<String, String> getMediationPolicyAttributes(String policyName, int tenantId, String direction,
+                                                                   APIIdentifier identifier) throws
+            APIManagementException {
+
+        org.wso2.carbon.registry.api.Collection seqCollection = null;
+        String seqCollectionPath = "";
+        Map<String, String> mediationPolicyAttributes = new HashMap<>(3);
+        try {
+            UserRegistry registry = ServiceReferenceHolder.getInstance().getRegistryService()
+                    .getGovernanceSystemRegistry(tenantId);
+
+            if ("in".equals(direction)) {
```

Comment:
Shall we use the constant APIConstants.API_CUSTOM_SEQUENCE_TYPE_IN here as well :)

Reply:
Fixed.